models/beers.js

```diff
@@ -0,0 +1,38 @@
+var keystone = require('keystone'),
+  Types = keystone.Field.Types;
+
+/**
+ * Beers Model
+ * ==========
+ */
+
+var Beers = new keystone.List('Beers', {
+  map: { name: 'beerName' },
+  autokey: { path: 'slug', from: 'beerName', unique: true }
+});
+
+Beers.add(  
+  {heading: 'Beer'},
+    {
+      beerTitle: { type: String },
+      beerName: { type: String, required: true, initial: true},
+      brewerName: { type: String, required: true, initial: true},
```

Comment:
You should make this a type of name. I assume that will make it easier for us to match ours together. 

Can we have a list of current Brewers and have that as a selection for this item?